mychanges/img/Readme

```diff
@@ -0,0 +1 @@
+Screenshots for my changes are stored here
```

Comment:
Are you trying to share screenshots of the changes? if so you can add them in the comments of your PR instead

Reply:
Ok,thanks!

Reply:
I am actually having trouble running the files to take the screenshots

Reply:
You can run your changes in Gitpod, it is similar to local environment